.github/workflows/tag.yml

```diff
@@ -0,0 +1,43 @@
+name: Set git tag
+
+on:
+  push:
+    branches: [main]
```

Comment:
This should run on the schedule that was previously in the docker workflow, not on pushes to main.

Reply:
When you push on main it should : Create a git tag -> trigger a docker build 
Right ?

Reply:
How it worked before is that there is a weekly schedule which creates the datestamped build for use in the Immich main branch, with pushes to main just building an image with a `:main` tag that went mostly unused.